public/angular/js/controllers/sources/form_modal.js

```diff
@@ -30,6 +30,7 @@ angular.module('Aggie')
           flash.setNotice('source.create.success');
           $rootScope.$state.go('source', { id: response._id }, { reload: true });
         }, function(err) {
+          console.log(err);
```

Comment:
Does this `console.log` have to be there? I'm guessing that it was added for debugging purposes, but no longer is needed.

Reply:
You are right :) Would remove.